docs/src/Pages/Guides/ReduxForm.example.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { DatePicker } from 'material-ui-pickers';
+import { Provider, connect } from 'react-redux';
+import { createStore, combineReducers } from 'redux';
+import { reduxForm, Field, getFormValues } from 'redux-form';
+import Code from '_shared/Code';
+import Grid from '@material-ui/core/Grid';
+import { reducer as formReducer } from 'redux-form';
+
+export const required = value => {
+  if (value && typeof value === 'string') {
+    value = value.trim();
+  }
+  return value ? undefined : 'Required';
+};
+
+export class ReduxDatePicker extends React.Component {
+  state = {
```

Comment:
But how are you getting this state on external validation? OnSubmit? It will submit anyway is there error or not?

Reply:
The ReduxDatePicker's state is for presentation purposes only. It displays the errors that come from DatePicker and the required method. The validation built into DatePicker only sets a value to the date prop when it's built in validation checks pass. Then the required method passed to Field on line 73 ensures that the field is populated to determine if the form may be submitted or not.

Reply:
This will not work for most users. I will create guide later

Reply:
Any news here? We are trying to get this done in a clean manner. An example or guide would be highly appreciated.

Reply:
bump.. any updates here?

Can you define "later"?